src/main/resources/tags/block_placer_ignored_materials.json

```diff
@@ -2,16 +2,60 @@
 	"values" : [
 		"#slimefun:sensitive_materials",
 		"#slimefun:mushrooms",
+		"#minecraft:tall_flowers",
 		"#minecraft:beds",
 		"#minecraft:doors",
-		"minecraft:sugar_cane",
 		"minecraft:bamboo",
-		"minecraft:cactus",
-		"minecraft:lily_pad",
-		"minecraft:dead_bush",
+		"#minecraft:rails",
+		"#minecraft:small_flowers",
+		"#minecraft:coral_blocks",
+		"#minecraft:coral_plants",
+		"#minecraft:small_flowers",
+		"#minecraft:buttons",
+		"#minecraft:banners",
+		"#minecraft:signs",
+		"#minecraft:standing_signs",
+		"#minecraft:wall_signs",
+		"#minecraft:wooden_pressure_plates",
```

Comment:
This is redundant, it's already inside #slimefun:sensitive_materials.

Reply:
isnt almost everything a sensitive material? like you stated in the comment on the tag 
> Things like Saplings or Pressure plates which break as well when you breakthe block beneath them.